### ISSUE TITLE
MeasuringTapeNode crosshairs have insufficient color contrast

### ISSUE THREAD

**Issue by arouinfar** [phetsims/energy-skate-park#468]
For #446. This issue likely should be transferred to scenery-phet, but opening in ESP for initial visibility.

At times, the red crosshairs can be hard to see due to low color contrast.

<img width="161" height="91" alt="Image" src="https://github.com/user-attachments/assets/edb34889-783d-49de-bb77-a115b4801f26" />

<img width="77" height="224" alt="Image" src="https://github.com/user-attachments/assets/a749d4f2-7c88-4af7-8500-88b6f98ee11a" />

For sims with a black background, the current color is fine, but black crosshairs would be a nice option to have in cases like ESP.

@samreid can we create an option for a black crosshairs?
